tools/kbs-client/src/main.rs

```diff
@@ -111,8 +111,18 @@ enum ConfigCommands {
     /// Set resource policy
     SetResourcePolicy {
         /// Policy file path
-        #[clap(long, value_parser)]
-        policy_file: PathBuf,
+        #[clap(long, value_parser, group = "resource_policy")]
+        policy_file: Option<PathBuf>,
+
+        /// Built-in policies
+        #[clap(long, action, group = "resource_policy")]
+        allow_all: bool,
+
+        #[clap(long, action, group = "resource_policy")]
+        deny_all: bool,
+
+        #[clap(long, action, group = "resource_policy")]
+        default: bool,
```

Comment:
Let's flourish the cmdline
```suggestion
        /// Use built-in policy: allow all accesses to KBS resource
        #[clap(long, action, group = "resource_policy")]
        allow_all: bool,
       
        /// Use built-in policy: deny all accesses to KBS resource
        #[clap(long, action, group = "resource_policy")]
        deny_all: bool,

        /// Use built-in policy: default policy (deny accesses from sample TEE)
        #[clap(long, action, group = "resource_policy")]
        default: bool,
```

Then the cmdline can be 
```
Set resource policy

Usage: kbs-client config --auth-private-key <AUTH_PRIVATE_KEY> set-resource-policy [OPTIONS]

Options:
      --policy-file <POLICY_FILE>  Policy file path
      --allow-all                  Use built-in policy: allow all accesses to KBS resource
      --deny-all                   Use built-in policy: deny all accesses to KBS resource
      --default                    Use built-in policy: default policy (deny accesses from sample TEE)
  -h, --help                       Print help
```